### ISSUE TITLE
About train loss

### ISSUE THREAD

**Issue by five-days** [huanglianghua/siamfc-pytorch#39]
train loss always 0.2 and 0.3 ,however,test can get a good result,why loss?

**Comment by wangchenw** [comment]
同问？

**Comment by conquerhuang** [comment]
The train process doesn't equal to tracking process. During the train process, there will be a lot of randomness introduced and result in different model. The train loss just reflect the fitting degree on train dataset not the same to the tracking performance on tracking benchmark. The more epochs we train the more models we will get. The best model not come into being in th last epoch. We just validation all the trained models on visual tracking benchmark and choose the best model  as the final model.
